ompi/mpi/fortran/mpif-h/Makefile.am

```diff
@@ -111,7 +111,7 @@ CLEANFILES += sizeof_f.f90
 # out this workaround indefinitely.  So Jeff advised Absoft to fix
 # this bug in Sep 2014.
 noinst_LTLIBRARIES=
-if BUILD_FORTRAN_SIZEOF
+if BUILD_PMPI_FORTRAN_MPIFH_BINDINGS_LAYER
```

Comment:
A few questions:

1. Shouldn't that be `MPI`, not `PMPI`?
1. Don't you also need the second conditional of `BUILD_FORTRAN_SIZEOF`?
1. Or, instead, should we make the `AM_CONDITIONAL` for `BUILD_FORTRAN_SIZEOF` only ever be true if `BUILD_*MPI_FORTRAN_MPIF_BINDINGS_LAYER` is also true?